### PR TITLE
Add masked background overlay to context chart, according to the design

### PR DESCRIPTION
**Pull request by stevenkampen**
screen shot 2018-12-10 at 14 55 25

@andrewthauer I'm not sure about the choice of white overlay here rather than a darker one. It might look better when I can put the more ornate handles and frame on this selection window.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "editor.formatOnSave": true,
```
Comment (andrewthauer) [inline .vscode/settings.json:5]: Why was this removed?

**Review by andrewthauer — APPROVED** [review]
@stevenkampen - I agree. Not sure I'm overly partial with the white, but your probably right about the handles making it more obvious.

File: .vscode/settings.json (hunk 1 above)
Comment (stevenkampen) [reply .vscode/settings.json:5]: @andrewthauer For me the auto-format is not adhering to the style conventions, so it was formatting everything incorrectly. I got sick of accidentally saving without that manually disabled and losing all my changes in a sea of formatting adjustments. I can fix it soon, but for the moment it's not worth the hassle.
